src/Nancy/StaticConfiguration.cs

```diff
@@ -105,6 +106,8 @@ var assembliesInDebug
         /// </summary>
         public static bool? DisableRequestStreamSwitching { get; set; }
 
+        public static bool AllowFileStreamUploadAsync { get; set; }

```

Comment:
I'd slap a comment on here to describe what will be affected by changing its value